examples/TestDriver/ios/Podfile.lock

```diff
@@ -55,7 +55,7 @@ PODS:
     - React/Core
     - React/fishhook
     - React/RCTBlob
-  - RNGestureHandler (1.0.17):
+  - RNGestureHandler (1.1.0):
```

Comment:
Ah, the version of RNGestureHandler (as dictated by `npm`, since we're just referring to the `.podspec` in `node_modules`) changed.  I remember we had some weirdness around that not being there, and we had added it explicitly at some point.  Not a big deal - feel free to check this in!